docs/getting-started/welcome.md

```diff
@@ -7,11 +7,9 @@ UMA stands for Universal Market Access. UMA is driven by a shared belief that fi
 
 UMA's goal is to enable anyone to attain or transfer any form of risk seamlessly and securely through UMA protocol to empower everyone to participate in a universally accessible financial system.
 
-If you would like to learn more about the concepts behind UMA's priceless financial contracts and oracle service, start by reading this [overview](getting-started/overview.md) of the UMA ecosystem.
-
 ## Developers
 
-If you are a developer interested in building on UMA Protocol, view this [explainer](synthetic-tokens/what-are-synthetic-assets.md) to understand how UMA's priceless synthetic tokens work. Also review the [development guide](build-walkthrough/build-process) to learn how to launch your own synthetic tokens on UMA. 
+If you are a developer interested in building on UMA Protocol, view this [explainer](synthetic-tokens/long-short-pair.md) to understand how UMA's Long Short Pair (LSP) contract works. Also review the [development guide](build-walkthrough/build-process) to learn how to launch your own financial contract on UMA. 
```

Comment:
I think this could use some more organization. We built the product sections, so let's use them now!

```suggestion
If you are a developer interested in building on the UMA Protocol, please refer to these tutorials and resources to get started.
- [Launching a KPI Option](https://docs.umaproject.org/kpi-options/summary)
- [Launching a Success Token](https://docs.umaproject.org/success-tokens/summary)
- [Integrating with the Optimistic Oracle](https://docs.umaproject.org/developers/optimistic-oracle-integration)
- [Proposing an UMA Improvement Proposal (UMIP)](https://docs.umaproject.org/uma-tokenholders/adding-price-id)
- [Understanding the Long Short Pair contract](https://docs.umaproject.org/synthetic-tokens/long-short-pair)
- [Understanding the Optimistic Oracle interface](https://docs.umaproject.org/oracle/optimistic-oracle-interface)
```